charts/prometheus-adapter/templates/certmanager.yaml

```diff
@@ -6,6 +6,13 @@ apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
   name: {{ template "k8s-prometheus-adapter.fullname" . }}-self-signed-issuer
+  {{- if .Values.customAnnotations }}
+  annotations:
+  {{- toYaml .Values.customAnnotations | nindent 4 }}
+  {{- end }}
+  labels:
+    app: {{ template "k8s-prometheus-adapter.name" . }}
```

Comment:
this is redundant with: "app.kubernetes.io/part-of"

Reply:
done.